resources/views/components/user/profile/moderation-overview.blade.php

```diff
@@ -0,0 +1,46 @@
+@props([
+    'targetUser' => null, // User
+])
+
+<div class="mb-6">
+    <div class="flex w-full items-center gap-x-1.5 mb-0.5">
+        <p role="heading" aria-level="2" class="text-2xs font-bold">
+            Moderation Overview
+        </p>
+    </div>
+
+    <div class="w-full p-2 bg-embed flex flex-col gap-y-2 rounded">
+        <div class="grid md:grid-cols-2 gap-x-12 gap-y-1">
+            @if ($targetUser->is_muted)
+                <x-user.profile.stat-element
+                    label="Muted until"
+                    value="{{ $targetUser->muted_until->format('Y-m-d') }}"
+                    :href="route('filament.admin.resources.users.edit', $targetUser)"
+                />
```

Comment:
I assume it's a timezone thing, but this is showing a the value from the database:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/5ff2a78a-c9ab-4789-90db-943df5f03eb7)

Which happens to be in the past, so it probably shouldn't be shown.

But the picker shows the previous day:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/30882d61-e637-4361-a40f-1db60dd4cd8e)

Reply:
Definitely seems to not be honoring the `is_muted` correctly:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/c52e4efc-51dd-48ab-8422-ccaa8d87522d)

Reply:
These are related to the manage page, not this element. I just searched for "Muted until".

Reply:
Should be resolved in latest:
* If `muted_until` is in the past, the value won't display in the Filament "view" panel.
* The datepicker now honors the UTC timezone for pre-populated values. This issue was caused by a client-side hydration issue with Filament's JS datepicker control.